docs/contributing/contributing-to-airbyte/README.md

```diff
@@ -4,45 +4,92 @@ description: 'We love contributions to Airbyte, big or small.'
+4. [Areas for contributing](https://docs.airbyte.io/v/dx-to-contribute_suggestion/contributing/contributing-to-airbyte#4-areas-for-contributing)
+5. [Ways you can contribute](https://docs.airbyte.io/v/dx-to-contribute_suggestion/contributing/contributing-to-airbyte#5-ways-you-can-contribute)
+6. [Review process](https://docs.airbyte.io/v/dx-to-contribute_suggestion/contributing/contributing-to-airbyte#6-review-process)
 
-**‌‌**If you are considering adding to the codebase or contributing a new integration, first a big thank you. We sincerely appreciate your help in our mission to solve the data integration problem!
+## 1. Code of conduct
+
+Please follow our [Code of conduct](https://docs.airbyte.io/contributing/code-of-conduct) in the context of any contributions made to Airbyte.
+
+## 2. The Airbyte specification
+
+Before you can start contributing, you need to understand [Airbyte's data protocol specification](https://docs.airbyte.io/architecture/airbyte-specification). 
+
+## 3. First-time contributors, welcome!
+
+We appreciate first time contributors and we are happy to assist you in getting started. In case of questions, just reach out to us via [email](mailto:hey@airbyte.io) or [Slack](https://slack.airbyte.io)!
 
 Here is a list of easy [good first issues](https://github.com/airbytehq/airbyte/labels/good%20first%20issue) to do.
 
-As soon as you are done with your development, just put up a PR.  
-When we review we look at:
+## 4. Areas for contributing
 
-* ‌Does the PR solve the issue?
-* Is the proposed solution reasonable?
-* Is it tested? \(unit tests or integration tests\)
-* Is it introducing security risks?
-* Does it pass QA?
+### **New integrations**
 
-‌Once your PR passes, we will merge it.
+It's easy to add your own integrations to Airbyte! Here are some links to instruct how to add sources and destinations. 
 
-## **Upvoting issues, feature and integration requests**
+#### **Contributing sources:**
 
-You are welcome to add your own reactions to the existing issues. We will take them in consideration in our prioritization efforts, especially for integrations.
+* [In Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/python-source/README.md)
+* [Based on Singer Taps in Python](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/singer-source/README.md)
 
-❤️ means that this task is CRITICAL to you.  
-👍 means it is important to you.
+#### **Contributing destinations:**
+
+* [In Java](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/java-destination/README.md)
+
+Please reach out to us if you'd like help developing integrations in other languages. **Indeed, you are able to create integrations using the language you want, as Airbyte connections run as Docker containers.** We haven't built the documentation for all languages yet, that's all. 
```

Comment:
can we move this section up closer to the suggestion? I think it's useful to say this upfront since the following links are language-specific.

Reply:
Made the change.